api/db/changelog/db.changelog-master.xml

```diff
@@ -31,5 +31,7 @@
   <include file="changelog/db.changelog-21.xml"/>
   <include file="changelog/db.changelog-22-email-verification-status.xml"/>
   <include file="changelog/db.changelog-23.xml"/>
+  <include file="changelog/db.changelog-24-email-verification-status-default.xml"/>
```

Comment:
Note: Had an interesting merge conflict here, as I had to rebase against change 25 and 24 is checked in but not yet added here. I tested locally that liquibase doesn't seem to care about ordering here (dev-up on master, then dev-up on this branch worked).